client/components/site-selector/style.scss

```diff
@@ -9,9 +9,6 @@
 	border: none;
 	z-index: z-index( 'root', '.site-selector' );
 
-	&.is-large .search  {
```

Comment:
Yes, that's correct. I suggest testing this with a user with only two sites or so, because the site selector changes. (It's why the border should not be applied to the sites element.)